Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/POSMessage.java

```diff
@@ -0,0 +1,32 @@
+package edu.asu.conceptpower.app.core;
+
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+
+@Component
+@PropertySource("classpath:pos.properties")
+public class POSMessage {
```

Comment:
You can get all POS from the interface that already exists and then get for each POS the corresponding message. You don't need an extra properties/messages file for POS values.